templates/cluster-template.yaml

```diff
@@ -29,6 +29,12 @@ spec:
     name: ${CLUSTER_IDENTITY_NAME}
   location: ${AZURE_LOCATION}
   networkSpec:
+    apiServerLB:
+      frontendIPs:
+      - name: ${CLUSTER_NAME}-api-lb
+        publicIP:
+          dnsName: ${CLUSTER_NAME}.${AZURE_LOCATION}.cloudapp.azure.com
```

Comment:
This is probably not going to be unique enough across all public DNS records for a given Azure region. We'll need an extra hash between "cluster name" and "location", probably.